src/Agenda.js

```diff
@@ -124,7 +131,7 @@ class Agenda extends React.Component {
       let title = get(event, titleAccessor);
 
       return (
-        <tr key={dayKey + '_' + idx} className={className} style={style}>
+        <tr key={dayKey + '_' + idx} className={className} style={style} onClick={()=>this.handleSelectEvent(day, events, dayKey)}>
```

Comment:
since this is only passing `events we don't need to use an arrow function here. you can right the `handleSelectEvent` method as `notify(this.props.onSelectEvent, this.props.events)`

Reply:
I don't understand.
Why retreive the whole events array, instead of just the selected event ?